src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/ApplicationsController.kt

```diff
@@ -347,10 +347,10 @@ class ApplicationsController(
     return applicationsTransformer.transformJpaToApi(application, offender, inmate)
   }
 
-  private fun getPersonDetailAndTransformToSummary(application: ApplicationEntity): ApplicationSummary {
-    val (offender, inmate) = getPersonDetail(application.crn)
+  private fun getPersonDetailAndTransformToSummary(application: uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApplicationSummary): ApplicationSummary {
```

Comment:
Do we need `uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApplicationSummary` here? Can we not import it with an alias?